### ISSUE TITLE
Issue with localization 

### ISSUE THREAD

**Issue by larynai** [Assistant/ModAssistant#180]
Some strings in program lost their localization 
![Screenshot_7](https://user-images.githubusercontent.com/44430804/83513111-ef6d5600-a4d9-11ea-96a4-d9b394d3cf92.png)

**Comment by larynai** [comment]
This problem has each localization
![Screenshot_8](https://user-images.githubusercontent.com/44430804/83513221-22174e80-a4da-11ea-9b46-08e25f4be9c5.png)

**Comment by Caeden117** [comment]
It looks like this problem is caused by the Intro page using `StaticResource`s instead of `DynamicResource`s

Here's an example from that Intro page.

```
<TextBlock
            Grid.Row="1"
            Margin="0"
            FontSize="28"
            FontWeight="Bold"
            Text="{DynamicResource Intro:Terms:Header}" />
<TextBlock
            Grid.Row="2"
            Margin="0,5"
            FontSize="16"
            TextWrapping="Wrap">
            <StaticResource ResourceKey="Intro:Terms:Line1" />
</TextBlock>
```
Each line of text with the `StaticResource` tag also happens to be the ones that are failing to localize, so its as easy as converting them over to `DynamicResource`.

I'll see if I cant quickly come up with a pull request.

**Comment by larynai** [comment]
Yes, I used last version (1.1.10)

**Comment by megalon** [comment]
This should be fixed with the latest release (1.1.11).
